### ISSUE TITLE
In a rare cases huge stock price can break the display

### ISSUE THREAD

**Issue by bjedrzejewski** [ScottLogic/StockFlux#563]
So, Berkshire Hathways is worth $200,000:

![image](https://cloud.githubusercontent.com/assets/9947570/13597589/553f6fc8-e511-11e5-9455-f4b19bef8ea2.png)

**Comment by owennw** [comment]
Well... yeah.

**Comment by owennw** [comment]
Either the tabs become wider, or the font size becomes smaller.
Either way an adaptable solution is preferred, so there's only an adjustment when needed.

**Comment by jackarnstein** [comment]
Just a suggestion, but you could use something like what BitFlux used for the volume values?

Numbers up to 1,000 are written as their full value, but then 1,000 gets shortened to 1k, 10,100 to 10.1k, 1,000,000 to 1m and so on...

**Comment by bjedrzejewski** [comment]
@jackarnstein I don't think this will work for this use case, as the small parts of the number (less than 1%) are often where the action happens.
